### PR TITLE
website additions

### PR DESCRIPTION
**Pull request by FrederikGielis**

### CODE REVIEW

File: webapp/src/MIlestone6.html [hunk 1]
```diff
@@ -5,32 +5,41 @@
...
+  <link rel="stylesheet" href="css/milestone.css">
   <title>Title</title>
 </head>
 <body>
 
 <header>
   <nav class="navbar navbar-expand-lg navbar-light">
     <div class="container-fluid">
-      <a href="#"><i class="fas fa-home fa-2x"></i></a>
+      <a href="/src/index.html"><i class="fas fa-home fa-2x"></i></a>
```
Comment (jonasclaes) [inline webapp/src/MIlestone6.html:16]: This will not work.

File: webapp/src/Milestone5.html [hunk 2]
```diff
@@ -1,39 +1,49 @@
...
 </head>
 <body>
 
 <header>
-  <nav class="navbar navbar-expand-lg navbar-light">
-    <div class="container-fluid">
-      <a href="#"><i class="fas fa-home fa-2x"></i></a>
+    <nav class="navbar navbar-expand-lg navbar-light">
+        <div class="container-fluid">
+            <a href="/src/index.html"><i class="fas fa-home fa-2x"></i></a>
```
Comment (jonasclaes) [inline webapp/src/Milestone5.html:16]: This will not work either.

**Review by jonasclaes — DISMISSED** [review]
Please fix the errors.

**Review by KamalidenovRustem — APPROVED** [review]

**Comment by KamalidenovRustem** [comment]
Please, push changes to main

**Pull request merged**